app/models/cloud_volume.rb

```diff
@@ -18,6 +19,14 @@ class CloudVolume < ApplicationRecord
 
   acts_as_miq_taggable
 
+  supports :cloud_volume_backup_create do
+    unsupported_reason_add(:cloud_volume_backup_create, _("Backup unsupported")) unless validate_backup_create
```

Comment:
@durandom ah, I see.  Does the code look okay now?

I renamed the method rather than rename the feature, as 'backup_create' seems a little too unspecific for a feature (as we may add backups to other objects besides volumes).